tests/Datagrid/PagerTest.php

```diff
@@ -32,6 +35,8 @@ public function entityClassDataProvider(): array
     }
 
     /**
+     * @param class-string $className
```

Comment:
```suggestion
     * @phpstan-param class-string $className
```
I guess `class-string` is not regular phpdoc, if not, there are others